### PR TITLE
Adding changes for issue #6303 for branch v2.0.x.

### PR DESCRIPTION
**Pull request by uberlinuxguy**
Signed-off-by: Jason Williams <uberlinuxguy@gmail.com>

### CODE REVIEW

**Comment by jsquyres** [comment]
Refs #6303

ok to test

**Comment by jsquyres** [comment]
@hppritcha Do we care about this for the v2.0.x branch?

**Comment by hppritcha** [comment]
@jsquyres i'm pretty sure this isn't going to compile right now owing to the ```<<<<<<```.  I'd say if @uberlinuxguy will fix up what appear to be traces of conflicts in the cherry-pick I'd say take it but no reason to do another release any time soon.

File: orte/mca/odls/alps/odls_alps_module.c [hunk 1]
```diff
@@ -365,6 +365,7 @@ static int close_open_file_descriptors(int write_fd, orte_iof_base_io_conf_t opt
     if (NULL == dir) {
         return ORTE_ERR_FILE_OPEN_FAILURE;
     }
+<<<<<<< HEAD
```
Comment (jsquyres) [inline orte/mca/odls/alps/odls_alps_module.c]: This won't compile.

**Comment by rhc54** [comment]
Sorry - I was blazing thru them all and didn't realize this was for 2.0.x. This isn't worth backporting that far. I'd just kill this PR and move on.

**Comment by jsquyres** [comment]
@rhc54 Roger. 

@uberlinuxguy Thanks for making all these PR's!  We're going to accept them for v2.1.x and beyond; we'll just close this v2.0.x one without merging.

**Comment by ibm-ompi** [comment]
The IBM CI (GNU Compiler) build failed! Please review the log, linked below.

 Gist: https://gist.github.com/ibm-ompi/88b480fb98dfd2bdac939f023e1f8fe6

**Comment by uberlinuxguy** [comment]
@jsquyres Yeah I was trying to merge it for the v2.x branch, guess I clicked on the wrong branch, but it was also required a bit more interaction to merge, the format is different from the other version.  I can fix the commit and re-submit a PR to the v2.x branch if you think it's worth it.

**Comment by jsquyres** [comment]
Yes, that would be great, thanks.  We didn't name that branch well -- v2.x is where v2.1.x is made from.

We almost certainly won't do another v2.1.x release, but the fix can be there on the branch / in the nightly snapshot in case anyone needs it.  Thank you!

**Pull request closed without merging**